app/views/accounts/show.html.erb

```diff
@@ -40,5 +40,14 @@
       },
       description: "Change your 2-step verification configuration."
     },
+    (
+      {
+        link: {
+          text: "Personal details",
+          path: account_personal_details_path,
+        },
+        description: "Make changes to your personal details such as your role.",
+      } if current_user.govuk_admin?
```

Comment:
Wouldn't it be better to use `if policy([:account, :personal_details]).show?` here?